torch/_inductor/mkldnn_lowerings.py

```diff
@@ -894,7 +902,26 @@ def qlinear_binary(
                     torch.tensor(x_scale, dtype=torch.float32), name="x_scale"
                 )
             else:
+                x_scale_size = x_scale.get_size()
+                if len(x_scale_size) == 2:
```

Comment:
Do we have the UT to cover this case for `qlinear_binary`?

Reply:
@leslie-fang-intel @chunyuan-w, to clarify, the change here is for fixing `qlinear_binary` lowering, which is unrelated to GEMM auto-tuning. So, it's not appropriate for me to add a UT for it in `test_cpu_select_algorithm.py`, since the GEMM template is currently not working for `qlinear_binary` with sum post-op right now. I'll add a UT in `test_mkldnn_pattern_matcher.py` instead. Thanks!